data/Gabriel-Giani.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "Gabriel Giani",
+  "bio": "I am a fullstack developer and a junior at Florida Atlantic University studying computer science." 
```

add a comma so the json will pass:
```suggestion
  "bio": "I am a fullstack developer and a junior at Florida Atlantic University studying computer science.",
```